apps/atomy-q/API/tests/Feature/Console/VerifyStorageDiskCommandTest.php

```
@@ -0,0 +1,87 @@
+            {
+                $this->contents = $contents;
+
+                return true;
+            }
+
+            public function get(string $path): string
+            {
+                return $this->contents;
+            }
+
+            public function delete(string $path): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$path'. (undefined)

(UnusedFormalParameter)